cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -54,6 +54,17 @@ describe('Top toolbar tests.', function() {
 			.should('exist');
 	});
 
+	it('Apply font size', function() {
+		cy.get('#tb_editbar_item_fontsizes')
+			.click();
+
+		cy.contains('ul', '48')
```

Comment:
'ul' means the whole list. What we need here is one list item. Which is not 'ul'. Also, you should use a more specific selector here instead of a general tag name. Like using a class in the selector.